lib/index.js

```diff
@@ -29,11 +29,6 @@ $(document).ready(() => {
     if (fileName === 'foods.html') {
       var foodId = event.target.parentElement.attributes.data.value.split('-')[1]
```

Comment:
This method of parsing up and manipulating the value of data to get the id works, another suggestion would be to use jQuery as a shorter refactor `$(event.target).parent().attr('data').split('-')[1]`.

Reply:
Not that much different, but just another way.